include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -353,6 +355,7 @@ struct Status : detail::Base<OrtStatus> {
   explicit Status(OrtStatus* status);      ///< Takes ownership of OrtStatus instance returned from the C API. Must be non-null
```

Comment:
> Must be non-null [](http://example.com/codeflow?start=110&length=17)

We can remote that restriction. An object holding nullptr_t would be considered as a success.

We may want to add IsOK() method, if we are to return Ort::Status from C++ API.